change/@ni-nimble-components-fdc2a0d0-af90-4afa-a47e-334c918f8566.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Opinion / not technical or impacting this PR. 

To me having the gap around the nav tree feels less solid / just like extra noise creating an unnecessary border:

![menuwithgap](https://github.com/user-attachments/assets/9fe478e2-b5d0-4e3b-9e9e-1f01f50644bb)

Compared to avoiding the gap and having the menu be flush with the edge of the page just feels more solid / structurally well composed / fit together / that lego snapped perfectly together fit

![menuwithoutgap](https://github.com/user-attachments/assets/ca0528e2-841c-4e80-a226-058c33e43771)

Reply:
My understanding is that this border is entirely intentional, but FYI @NIbokeefe.